src/js/main.js

```diff
@@ -108,8 +115,34 @@ var SupportKit = Marionette.Object.extend({
                     id: res.appUserId
                 });
 
+                var EventCollection = BaseCollection.extend({
+                    url: urljoin('appusers', res.appUserId, 'event'),
+                    model: Event
+                });
+
+                var RuleCollection = Backbone.Collection.extend({
+                    model: Rule
+                });
+
+                // this._events overrides some internals for event bindings in Backbone
+                this._eventCollection = new EventCollection(res.events, {
```

Comment:
By swapped out, you mean resetted?

There's a `destroy` method now on the SupportKit object that nukes everything (collection, models, events, and the widget itself) and it can be re-init after that. Why would you re-init SupportKit without destroying it first?